lib/src/main/java/com/ibm/bluemix/appid/android/api/LoginWidget.java

```diff
@@ -17,8 +17,13 @@
 import android.support.annotation.NonNull;
 
 public interface LoginWidget {
-	// TODO: What is the usecase for this method?
-//	void launch (@NonNull final Activity activity, @NonNull final AuthorizationListener authorizationListener, String accessTokenString);
+	/**
+	 * Lunch the login widget user interface
+	 * @param activity Parent activity
+	 * @param authorizationListener
+	 * @param accessTokenString
+	 */
+	void launch (@NonNull final Activity activity, @NonNull final AuthorizationListener authorizationListener, String accessTokenString);
```

Comment:
1. This is a behavior that is supported by the REST API and also on the Swift SDK, so we need to be consistent and support it on Android
2. There is no reason to not allow this, its a capability we get for free
3. Use cases:
a. an application flow where the developer allows multiple anonymous logins and then wants to choose the one he wants to upgrade to identified, not necessarily the last one.
b. the developer wants to store away the anonymous token and use it in the upgrade if the application got restarted before the user logged in with identity.